### ISSUE TITLE
Need to add some github-actions

### ISSUE THREAD

**Issue by yashsehgal** [DesignSystemsOSS/getbuttons.io#10]
Some basic Github Actions needs to be added

- Stale Bot
- Husky to run ESLint at pre-commits and pre-pull-requests
- DeepSource (https://deepsource.io/)
- Welcome and Greetings Bot

**Comment by SamikshaSankar** [comment]
I would like to work on this issue as well as love to learn something new
please assign me @yashsehgal

**Comment by yashsehgal** [comment]
Thanks for showing your interest. Definitely, You can get started with this one.

**Comment by kanak22** [comment]
@SamikshaSankar any updates?

**Comment by SamikshaSankar** [comment]
Yup... Actually my exams going on...before 22jun I will be complete this task

**Comment by jayant766** [comment]
please assign it to me under LGM-SOC'21

**Comment by yashsehgal** [comment]
@SamikshaSankar Are you able to work on this issue, If not then it's fine, I can assign this issue to @jayant766

**Comment by kanak22** [comment]
@SamikshaSankar are u working on it? its 21 today .. u said u will do it before 22nd ... kindly complete the issue by tomorrow or else it will go to @jayant766

**Comment by SamikshaSankar** [comment]
@kanak22 yes u can assign to @jayant766

**Comment by kanak22** [comment]
@jayant766 this issues goes to you,start working on it ... you have 5 days to complete it :)

**Comment by RituCs** [comment]
I would like to work on this issue.

**Comment by kanak22** [comment]
@jayant766 it's been 8 days u haven't gave any updates on this issue

**Comment by yashsehgal** [comment]
@kanak22 Thanks for handling things very actively, This issue has to be re-assigned to @RituCs 👍🏽

**Comment by kanak22** [comment]
@RituCs  any updates?

**Comment by RituCs** [comment]
> @RituCs any updates?

Sorry for the delay. I was busy for final sem exam.From tomorrow I will start working.Please give some time.Thanks!

**Comment by kanak22** [comment]
okay u have 4 days :)
